OSSMDOC-303: Config reorg (#32055)
openshift/openshift-docs: modules/ossm-config-web-console.adoc

// Module included in the following assemblies:
//
// * service_mesh/v1x/customizing-installation-ossm.adoc
// * service_mesh/v2x/customizing-installation-ossm.adoc

[id="ossm-config-web-console_{context}"]
= Configuring the control plane with the web console

> Follow this procedure to configure the `ServiceMeshControlPlane` by using the {product-title} web console.

Reviewer: ```suggestion
You can configure the `ServiceMeshControlPlane` by using the {product-title} web console.
```

Reviewer: Avoid self-referential text whenever possible, such as "This topic is about" or "This section describes".
Instead, focus on the purpose or value of the content, and include words that users are likely to search
for. For example, in a short description in an online help topic, provide users with enough information to
determine whether the topic meets their needs. You can use self-referential text in learning modules
and tutorials.

Reviewer: Multiple instances of "Follow this procedure" and "this section."

Author: @rolfedh Removed a bunch of instances of "follow this procedure." I left a couple of instances and and a this section that will be changed in a subsequent PR.

.Procedure

. In the {product-title} web console, click *Operators* -> *Installed Operators*.

. Select the project where you installed the control plane, for example `istio-system`, from the Project menu.

. Click the {ProductName} Operator. In the *Istio Service Mesh Control Plane* column, click the name of your `ServiceMeshControlPlane` resource, for example `basic`.

> . On the *Details* page, click the toggle 

Reviewer: ```suggestion
. On the *Details* page, click the toggle.
```
